### DISCUSSION TITLE
使用激进的`sidebar: "structure"`,没有生成期望的sidebar

### DISCUSSION THREAD

**Discussion by yzqdev** [vuepress-theme-hope/vuepress-theme-hope#1593] [Q&A] [answered]
我按照文档链接[链接](https://vuepress-theme-hope.github.io/v2/zh/guide/layout/sidebar.html#:~:text=%E5%A6%82%E6%9E%9C%E4%BD%A0%E4%BD%BF%E7%94%A8%E7%BB%93%E6%9E%84%E7%94%9F%E6%88%90%E7%9A%84%E6%96%87%E4%BB%B6%E5%A4%B9%E4%B8%8B%E5%B5%8C%E5%A5%97%E4%BA%86%E5%85%B6%E4%BB%96%E6%96%87%E4%BB%B6%E5%A4%B9%E4%B8%94%E6%96%87%E4%BB%B6%E5%A4%B9%E5%8C%85%E5%90%AB%20README.md%20%E6%96%87%E4%BB%B6%EF%BC%8C%E5%88%99%E5%AF%B9%E5%BA%94%E7%9A%84%E6%96%87%E4%BB%B6%E5%A4%B9%E4%BC%9A%E8%A2%AB%E6%B8%B2%E6%9F%93%E6%88%90%E4%B8%80%E4%B8%AA%E5%88%86%E7%BB%84%E3%80%82%E6%89%80%E4%BB%A5%E4%BD%A0%E7%94%9A%E8%87%B3%E5%8F%AF%E4%BB%A5%E6%9B%B4%E5%8A%A0%E6%BF%80%E8%BF%9B%EF%BC%8C%E6%AF%94%E5%A6%82%E7%9B%B4%E6%8E%A5%E8%AE%BE%E7%BD%AE%20sidebar%3A%20%22structure%22%20%E8%AE%A9%E4%BD%A0%E7%9A%84%E4%BE%A7%E8%BE%B9%E6%A0%8F%E5%85%A8%E9%83%A8%E4%BB%8E%E6%96%87%E4%BB%B6%E7%BB%93%E6%9E%84%E4%B8%AD%E8%87%AA%E5%8A%A8%E7%94%9F%E6%88%90%E3%80%82)配置了`sidebar:'structure'`,可是似乎没有生成sidebar

我的仓库地址: https://github.com/yzqbugs/hope-demo.git
演示地址: https://yzqbugs.github.io/hope-demo/guide/

不知道是我配置的问题呢,还是主题的问题呢?

**Comment by Mister-Hope** [comment]
If you are creating from template and not working well, that should be a bug.

Probably need some investigation here, I will update when I have time.

**Comment by Mister-Hope** [comment, marked as answer]
There are some issues dealing with themeConfig with no `locales`, and it has been fixed in https://github.com/vuepress-theme-hope/vuepress-theme-hope/commit/34306394ac89a2301dc4af09eed4da2ae1c14fe8

So beta.21 probably solves your problem

**Reply by yzqdev** [reply]
Great! Thank you
